contentcuration/contentcuration/assets/shared/views/ThumbnailUpload.vue

```diff
@@ -0,0 +1,135 @@
+<template>
```

Comment:
Rather than work out how to use dropzone in Vue JS, or get quagmired in integrating uppy, this is a quick thumbnail uploader component that is drag and drop compatible (uses native drag and drop which is compatible with all of the browsers that we support).

Reply:
Nice! It would be great to reuse this for other kinds of file uploads